Doc/library/os.rst

```diff
@@ -217,7 +217,8 @@ process and user.
 .. function:: getenv(key, default=None)
 
    Return the value of the environment variable *key* if it exists, or
-   *default* if it doesn't. *key*, *default* and the result are str.
+   *default* if it doesn't. Only *key* needs to be a string as it is used to
```

Comment:
Thanks @dorosch for improving this!

I think we should use `*key* must be a string.`. The "Only" does not make sense if you don't know the previous phrasing and "as it is used to lookup the value in os.environ." is an implementation detail that is not relevant here.

Reply:
Thanks for review @remilapeyre 
Great point, I'll fix it.